### DISCUSSION TITLE
5.0 公众号返回链接消息报错

### DISCUSSION THREAD

**Discussion by tgpx1899** [w7corp/easywechat#2597] [Q&A]
[0]Class "EasyWeChat\Kernel\Messages\Message" cannot support transform to XML message.

**Comment by overtrue** [comment]
贴你的代码……

**Comment by overtrue** [comment]
https://developers.weixin.qq.com/doc/offiaccount/Message_Management/Passive_user_reply_message.html

> 开发者可以在响应包（Get）中返回特定 XML 结构，来对该消息进行响应（现支持回复文本、图片、图文、语音、视频、音乐）

没有你说的链接消息呢。
